core/kernel/Complex_spec.rb

```diff
@@ -269,4 +269,8 @@
       end
     end
   end
+
+  it "freezes its result" do
```

Comment:
Maybe a bit confusing since it might just be a property of all instances of that class, i.e., they might be "created frozen", but I think it's OK as-is, up to the implementation to decide how to do it